### ISSUE TITLE
use of pip module means "--check" always shows `changed`

### ISSUE THREAD

**Issue by kennethgillen** [openmicroscopy/prod-playbooks#21]
use of pip module means "--check"mode always shows `changed` for pip tasks.

Internal OME links: 
 https://openmicroscopy.slack.com/archives/C0K5ME1EW/p1512749355000430 
 https://github.com/openmicroscopy/management_tools/pull/612

**Comment by manics** [comment]
Has anyone tested with the latest ansible 2.4.2 (or even 2.4.3.0-0.1.beta1)?

**Comment by manics** [comment]
Also is this an issue only with check mode (so when running for real you **don't** get incorrect statuses or handlers being trigger)?

**Comment by kennethgillen** [comment]
@manics re: your last: I haven't re-run since the demo upgrade (which had to restart web anyway) so can't say. We'll have to make a note next time we run these.

**Comment by sbesson** [comment]
My recent run of the demo-server using Ansible 2.3.1.0 without the check mode showed no changed state for the `openmicroscopy.omero-web : omero web | setup virtualenv` task.

**Comment by manics** [comment]
Is this still relevant?

**Comment by joshmoore** [comment]
Yes. I'm certainly still periodically seeing `changed` for pip lines when using check mode
